### PR TITLE
sqlite-utils: add page

### PR DESCRIPTION
**Pull request by jeffabailey**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

3.26

### CODE REVIEW

File: pages/common/sqlite-utils.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# sqlite-utils
+
+> Command-line tool used to manipulate SQLite databases in a number of different ways.
+> More information: <https://sqlite-utils.datasette.io/en/stable/cli.html>.
+
+- Create a database:
+
+`sqlite-utils create-database person.db`
```
Comment (owenvoke) [inline pages/common/sqlite-utils.md:8]:
```suggestion
`sqlite-utils create-database {{path/to/database.db}}`
```

File: pages/common/sqlite-utils.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Create a table:
+
+`sqlite-utils create-table person.db people id integer name text height float photo blob --pk id`
```
Comment (owenvoke) [inline pages/common/sqlite-utils.md:12]:
```suggestion
`sqlite-utils create-table {{path/to/database.db}} {{table_name}} {{id integer name text height float photo blob --pk id}}`
```

File: pages/common/sqlite-utils.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Upsert a record:
+
+`echo '[ {"id": 1, "name": "Linus Torvalds"}, {"id": 2, "name": "Steve Wozniak"}, {"id": 3, "name": "Tony Hoare"} ]' | sqlite-utils upsert person.db people - --pk id`
```
Comment (owenvoke) [inline pages/common/sqlite-utils.md:16]:
```suggestion
`{{echo '[ {"id": 1, "name": "Linus Torvalds"}, {"id": 2, "name": "Steve Wozniak"}, {"id": 3, "name": "Tony Hoare"} ]'}} | sqlite-utils upsert {{path/to/database.db}} {{table_name}} - {{--pk id}}`
```

File: pages/common/sqlite-utils.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Select records:
+
+`sqlite-utils rows person.db people`
```
Comment (owenvoke) [inline pages/common/sqlite-utils.md:20]:
```suggestion
`sqlite-utils rows {{path/to/database.db}} {{table_name}}`
```

File: pages/common/sqlite-utils.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Delete a record:
+
+`sqlite-utils query person.db "delete from people where name = 'Tony Hoare'"`
```
Comment (owenvoke) [inline pages/common/sqlite-utils.md:24]:
```suggestion
`sqlite-utils query {{path/to/database.db}} {{"delete from people where name = 'Tony Hoare'"}}`
```

File: pages/common/sqlite-utils.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Drop a table:
+
+`sqlite-utils drop-table person.db people`
```
Comment (owenvoke) [inline pages/common/sqlite-utils.md:28]:
```suggestion
`sqlite-utils drop-table {{path/to/database.db}} {{table_name}}`
```

**Review by owenvoke — CHANGES_REQUESTED** [review]
Hi @jeffabailey, just a couple of tweaks you can make to add token syntax. 👍🏻

**Review by owenvoke — APPROVED** [review]

File: pages/common/sqlite-utils.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+`{{echo '[ {"id": 1, "name": "Linus Torvalds"}, {"id": 2, "name": "Steve Wozniak"}, {"id": 3, "name": "Tony Hoare"} ]'}} | sqlite-utils upsert {{path/to/database.db}} {{table_name}} - {{--pk id}}`
+
+- Select records:
+
+`sqlite-utils rows {{path/to/database.db}} {{table_name}}`
+
+- Delete a record:
+
+`sqlite-utils query {{path/to/database.db}} {{"delete from {{table_name}} where name = 'Tony Hoare'"}}`
```
Comment (navarroaxel) [inline pages/common/sqlite-utils.md:24]: By convention we use quotes `"` outside the token syntax. Also, the token syntax can't be nested, it's a markdown spec
```suggestion
`sqlite-utils query {{path/to/database.db}} "{{delete from table_name where name = 'Tony Hoare'}}"`
```

Comment (navarroaxel) [reply pages/common/sqlite-utils.md:24]: I'll apply this quick fix, then merge.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**